packages/components/ng-ovh-contracts/src/summary/component.js

```diff
@@ -3,6 +3,7 @@ import template from './template.html';
 const component = {
   bindings: {
     contracts: '<items',
+    inlineContracts: '<inlineItems',
```

Comment:
```suggestion
    contracts: '<',
    inlineContracts: '<',
```

**suggestion:** Could be better to use the same labels

Reply:
Yep, it could be better.
Doing this, it means i have to refactor all places where we use the `ovh-contracts` module.

And regarding goals of my current mission, it's not suitable to spent time on that.
Probably duable in normal sprint.

Reply:
@mohammed-zahaf I don't understand this is a property you're adding right  ? So you could name it as you want

Reply:
Yes, i add the `inlineContracts: '<inlineItems',` to have an ISO naming like for `contracts: '<items'`.